src/Utils.php

```diff
@@ -14,14 +14,14 @@ final class Utils
     /**
      * Remove the items given by the keys, case insensitively from the data.
      *
-     * @param string[] $keys
+     * @param (string|int)[] $keys
```

Comment:
please revert all changes to this. we know this is an issue, and the phpdoc intentionally does not allow ints.

Reply:
> 

Oh, I see. It looks like designing for string `keys` removal only. Let me revert it.